modules/ROOT/pages/metrics/custom-metrics-connector.adoc

```diff
@@ -0,0 +1,96 @@
+= Anypoint Custom Metrics Connector
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using Custom Dashboards. 
+
+Once the metrics are in a Custom Dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
+
+[NOTE] 
+====
+To use the Anypoint Custom Metrics connector, you must have a Titanium subscription. +
+This feature is currently supported only for hybrid applications.
+====
+
+== Prerequistes
+
+* Mule runtime engine 4.1.x
+* xref:monitoring::am-installing.adoc[Install the Anypoint Monitoring Agent]
+
+== Configure the Custom Metrics Values Using Design Center
+
+. Search Anypoint Exchange for the Anypoint Custom Metrics connector. 
+
+. Select the trigger for your flow.
+. Select the *Anypoint Custom Metrics Connector* as the *Target* in your flow, and click *Done*. 
++
+image::metrics/custom-metric-dc-select-target.png[]
++
+The connector has one operation called "Send custom metric." 
+. On the design canvas, open the Anypoint Custom Metrics Connector card to add your metrics.
+. In *Metric Name*, enter a meaningful value. 
+. Next to *Enter Contextual Information*, click *Add* to enter the name and value for the metric's dimension.  
++
+image::metrics/custom-metric-dc-add-dimension.png[]
+.. In the *Add Item* dialog box, enter the *Name* and *Value* for the dimension.
+.. Leave the *Collapsible* option with the default of *True*.
+.. Click *OK*. 
++
+image::metrics/custom-metric-dc-add-item.png[]
++
+You can repeat these steps to add multiple key values.
+.  Next to *Enter Measurements*, click *Add* to add facts for the metric. Values for measurements are always numerical.
+.. In the *Add Item* dialog box, enter a *Name* and numerical *Value* for the fact. +
+[NOTE]
+For facts, the value must always be numerical. This is not validated at design time, but it will cause a runtime exception once deployed.
+.. Leave the *Collapsible* option with the default of *True*.
+.. Click *Finish*. +
+You can repeat these steps to add multiple key values.
+. Select the *Throw Runtime Exception on Validation Failure* option if you want the flow to break if invalid values are entered for the metric. If this option is left unchecked, an error will be logged, but the flow won't break.
+
+
+== Configure Custom Metrics Values Using Anypoint Studio
+
+. Search Anypoint Exchange for the Anypoint Custom Metrics connector. +
+The connector has one operation called "Send custom metric." 
+. Add the *Send Custom Metric* operation to your application flow. +
```

Comment:
Exchange in Studio does not let one add an operation. Users must click "Add to project" to add the connector to their project. Please discuss this. The operation only is relevant after adding the connector to the project and you drag the operation to the canvas.